src/app/sections/ProbeDetails.svelte

```diff
@@ -149,18 +158,12 @@ h2 {
     <!-- probe-details-content -->
     <div class="probe-details-content">
         {#if $store.probe.type}
-            <div class="drawer-section probe-labels">
-                <div>
-                    <span
-                        style="display: inline-block;"
-                        class="label label-text--01 label--{$store.probe.type}">{$store.probe.type}</span>
-                </div>
+            <dl class="drawer-section info-type">
+                <dt>{$store.probe.type}</dt>
```

Comment:
the `dt` here looks a little similar to the headers in the other parts of the details section. Could we play with making it not using uppercase + making it a bit bigger?